### ISSUE TITLE
Cannot read property 'query' of undefined (wrapStore.js:166)

### ISSUE THREAD

**Issue by salnikovp0** [tshaddix/webext-redux#203]
Hi,

Ive updated the version to the latest 2.0.2, And i started to get this Error when i try to load a page that content.html.
The manifest.json doesn't changed, got the Permission: tabs included like before.

**Comment by tshaddix** [comment]
Hello @salnikovp0 - Can you give us a little more detail around the structure of your app? Are you including `webext-redux` in a normal html page?

**Comment by salnikovp0** [comment]
Yes it happens from the version of "react-chrome-redux": "2.0.0-alpha.5" and newer.
I use it also in a content page, when declare the
"content_scripts": [
    {
      "matches": ["*://*"],
      "js": ["pages/index.js"],
      "run_at": "document_end"
    }
  ]
It try's to load the webapi in the html page, and not in the background side. that's why it falls

**Comment by tshaddix** [comment]
Got it - `wrapStore` was not designed to be used on a content script. Is there a reason you are using `wrapStore` on a content-script rather than a Proxy Store?

**Comment by salnikovp0** [comment]
I am using the "this.props" in order to get data from the state. Just like
simple react application

בתאריך שבת, 13 באפר׳ 2019, 21:59, מאת Tyler Shaddix ‏<
notifications@github.com>:

**Comment by tshaddix** [comment]
@salnikovp0 Is there any way you can share a snippet of your code for your content script that's throwing the error?
